src/components/Article/ArticleCard.js

```diff
@@ -25,7 +25,7 @@ const ArticleCard = ({ ...props }) => {
         <div className="description">
           {author.username}
         </div>
-        
+
```

Comment:
why the blank line?

Reply:
I don't think it has any, let me remove it.